assets/src/apps/authoring/Accordion/Accordion.tsx

```diff
@@ -1,37 +1,53 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable react/jsx-key */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable react/prop-types */
 
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+const plusIcon = require('../../../../static/images/plusIcon.svg');
```

Comment:
not seeing where this is used, also I think things in the static folder should be accessible via url as a hosted item, so I don't think you'd need to require them

Reply:
I have a branch where I am working on bringing the data in for the authoring tool, here I have added a way for React to access the path to the static folder correctly
https://github.com/Simon-Initiative/oli-torus/blob/d28837fc162998a40d8f6d6b48ac236212d71818/lib/oli_web/templates/resource/advanced.html.eex

this is the branch: https://github.com/Simon-Initiative/oli-torus/tree/feature/authoring-bff

so I think what we should do is after bringing that path into React props then just use 
```
<img src={`${imagePath}/plusIcon.svg`}
```
something like that. (the path would be in `props.paths.images`) . 
it's also possible that you might want to create a component like `<Icon basePath={props.paths.images} file="plusIcon.svg" />`  and then internally render the `img` tag with maybe a fallback if the image isn't found or something. Possibly also have a map of icon names and / or accept a url.

You might want to build off of my branch and then we can just merge the changes together tomorrow.

Reply:
This line i forgot to remove. I have removed that now. But what you have explained i can use that for header tab icons.